### PR TITLE
Skeleton for the Wasm object

### PR DESCRIPTION
**Pull request by jfbastien**
We should document this better. Here's a start.

### CODE REVIEW

**Comment by jfbastien** [comment]
I believe @lukewagner volunteered to document this further :D

**Comment by s3ththompson** [comment]
Looks like a good start. I believe this will eventually deserve its own file in the repo (and we should liberally add code examples, etc.)

**Comment by dschuff** [comment]
I guess this is in the same category as the ABI, as in "not part of the core WebAssembly spec but important for implementors and/or users". Should we maybe create a special directory in the design repo for those things? what else is in that category?

**Comment by jfbastien** [comment]
Let's start small with this, and create the One True Description as we figure things out.

File: Web.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ than if the module was JavaScript.
 More concretely, the following is a list of points of contact between WebAssembly
 and the rest of the Web platform that have been considered:
 
+## The `Wasm` object
+
+WebAssembly is exposed to the Web through a `Wasm` object, whose API is designed
```
Comment (lukewagner) [inline Web.md]: This isn't the only way Wasm is exposed to the Web (or even JS).  With ES6 module integration, there'd also be <script type=module> as well import from ES6.  Also, with Web API integration, one should be able to pass the URL of a .wasm to a Worker constructor.

Comment (binji) [inline Web.md]: ...the suspense is killing me...

Comment (jfbastien) [inline Web.md]: Must be using Streams API, got a backpressure signal :(

Comment (lukewagner) [inline Web.md]: Hah, so weird, I distinctly remember writing a longer comment.  Oh I know what happened, I wrote &lt;script&gt; without escaping the &lt;.  Sorry about that.

I meant to end with: ... WebAssembly exposed through &lt;script type=module&gt;, `import`, and Worker constructors that took a module URL.

Comment (jfbastien) [inline Web.md]: Would you rather I simply rephrase as "Besides [modules](LINK), WebAssembly is exposed to the Web through..." ?

That should be in its own section IMO, and currently isn't even A Thing.

Comment (jfbastien) [inline Web.md]: Ping.

Comment (lukewagner) [inline Web.md]: (Sorry, done with GDC crazy-times now.)  Yes, that makes sense.  How about "In addition to [ES6 Modules] integration, WebAssembly is exposed to JavaScript through..."  That'd lgtm

Comment (jfbastien) [inline Web.md]: Done, ty.

**Comment by jfbastien** [comment]
Got @lukewagner's (now-hidden) lgtm here: https://github.com/WebAssembly/design/pull/610#discussion_r56727950, merging.

**Pull request merged**